src/testIntegration/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/IntegrationTest.kt

```diff
@@ -217,11 +217,10 @@ abstract class IntegrationTest {
         registry.add("mailcatcher.api.port") { getMappedPort(1080) }
       }
       localStackContainer?.run {
-        getEndpointConfiguration(S3)
-          .run {
-            registry.add("app.s3.localstack-url") { serviceEndpoint }
-            registry.add("app.s3.region") { signingRegion }
-          }
+        registry.add("app.s3.localstack-url") {
```

Comment:
Integration tests were failing on dev. Changed this based on code changes in visit-scheduler.